contracts/helper/RebalancingSetCTokenBidder.sol

```diff
@@ -0,0 +1,360 @@
+    /*
+    Copyright 2020 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+pragma solidity 0.5.7;
+pragma experimental "ABIEncoderV2";
+
+import { ReentrancyGuard } from "openzeppelin-solidity/contracts/utils/ReentrancyGuard.sol";
+import { SafeMath } from "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+import { CommonMath } from "../lib/CommonMath.sol";
+import { IRebalanceAuctionModule } from "../core/interfaces/IRebalanceAuctionModule.sol";
+import { IRebalancingSetToken } from "../core/interfaces/IRebalancingSetToken.sol";
+import { ITransferProxy } from "../core/interfaces/ITransferProxy.sol";
+import { ICToken } from "./interfaces/ICToken.sol";
+
+import { ERC20Wrapper } from "../lib/ERC20Wrapper.sol";
+
+
+/**
+ * @title RebalancingSetCTokenBidder
+ * @author Set Protocol
+ *
+ * A helper contract that mints a cToken from its underlying or redeems a cToken into 
+ * its underlying used for bidding in the RebalanceAuctionModule.
+ */
+contract RebalancingSetCTokenBidder is
+    ReentrancyGuard
+{
+    using SafeMath for uint256;
+    
+    // Address and instance of RebalanceAuctionModule contract
+    IRebalanceAuctionModule public rebalanceAuctionModule;
+
+    // Address and instance of TransferProxy contract
+    ITransferProxy public transferProxy;
+
+    // Address and instance of CToken contract
+    ICToken public targetCToken;
```

Comment:
If we want to reuse this contract, we can create a simple mapping of underlying -> cToken.